tools/get_vale_styles.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# Fail on errors and display commands
+set -ex
+
+# Get fresh Vale styles
+cd .vale/styles || exit
```

Comment:
maybe store styles in the docker image as for che-blog ?

It may avoid to pull / unpack zip files on every preview command

Reply:
The lifecycle of the styles is currently much shorter, we want to have the latest styles without needing to rebuild the container.

Reply:
Currently we can have multiple releases a week for both styles. Hopefully they will stabilize soon.